packages/world-modules/mud.config.ts

```diff
@@ -131,42 +131,39 @@ export default mudConfig({
       },
       tableIdArgument: true,
     },
-    /************************************************************************
-     *
-     *    ERC20 MODULE
-     *
-     ************************************************************************/
-    ERC20Metadata: {
-      directory: "modules/erc20-puppet/tables",
-      keySchema: {},
+    TokenURI: {
+      directory: "modules/tokens/tables",
+      keySchema: {
+        tokenId: "uint256",
+      },
       valueSchema: {
-        decimals: "uint8",
-        name: "string",
-        symbol: "string",
+        tokenURI: "string",
       },
       tableIdArgument: true,
     },
-    Allowances: {
-      directory: "modules/erc20-puppet/tables",
+    OperatorApproval: {
+      directory: "modules/tokens/tables",
       keySchema: {
-        account: "address",
-        spender: "address",
+        owner: "address",
+        operator: "address",
       },
       valueSchema: {
-        value: "uint256",
+        approved: "bool",
       },
       tableIdArgument: true,
     },
-    TotalSupply: {
-      directory: "modules/erc20-puppet/tables",
+    Metadata: {
```

Comment:
thoughts on these tables using a more specific name like `TokenMetadata` and `TokenRegistry`?

also would be nice to pull these non-ERC1155 changes/refactors out into a separate PR that preps for this one so that the diff here is smaller

Reply:
Sounds good, I'll revert the changes made to ERC20 and ERC721 puppets and prep the way for the refactor PR

Reply:
fixed in [eba27be](https://github.com/latticexyz/mud/pull/2198/commits/eba27beb1823a7bec70f362bb330f5a0781ccc87) 👍